app/component/CustomizeSearch.js

```diff
@@ -67,28 +71,60 @@ class CustomizeSearch extends React.Component {
   }
 
   componentWillMount() {
+    const custSettings = getCustomizedSettings();
+
     this.walkReluctanceSliderValues =
       CustomizeSearch.getSliderStepsArray(0.8, 10, 2).reverse();
-    this.walkReluctanceInitVal = this.context.location.query.walkReluctance ?
-      mapToSlider(this.context.location.query.walkReluctance, this.walkReluctanceSliderValues) :
-      10;
+    if (custSettings.walkReluctance) {
+      this.walkReluctanceInitVal = custSettings.walkReluctance
+      && mapToSlider(custSettings.walkReluctance, this.walkReluctanceSliderValues);
+    } else if (this.context.location.query.walkReluctance) {
+      this.walkReluctanceInitVal = this.context.location.query.walkReluctance
+      && mapToSlider(this.context.location.query.walkReluctance, this.walkReluctanceSliderValues);
+    } else {
+      this.walkReluctanceInitVal = 10;
+    }
```

Comment:
Should the url parameters have highest priority (here and for other sliders, too)? In other words,m the first test is 
   if (this.context.location.query.walkReluctance) { ...
and then comes 
   else if if (custSettings.walkReluctance) { ...